test/pkg.jl

```diff
@@ -75,6 +80,10 @@ temp_pkg_dir() do
     @test isempty(Pkg.installed())
     @test !isempty(Pkg.available("Example"))
     @test !in("Example", keys(Pkg.installed()))
+    Pkg.rm("Example") #second rm to make sure things are ok
```

Comment:
Testing [this](https://codecov.io/github/kshyatt/julia/base/pkg/entry.jl?ref=5e01b1ab5bfafe8e84c2569b19d43952e689dcda#l-82) (make sure nothing explodes horribly)